ads_plugin.py

```diff
@@ -610,7 +605,7 @@ def on_fill_axons_button(self, event):
 
         axon_corr_array = np.flipud(axon_extracted_array)
         axon_corr_array = params.intensity['binary'] * np.rot90(axon_corr_array, k=1, axes=(1, 0))
-        file_name = self.ads_temp_dir / (myelin_mask_overlay.name[:-len("-myelin")] + "-axon-corr.png")
+        file_name = self.ads_temp_dir / (myelin_mask_overlay.name[:-len("-myelin-manual_pred")] + "-axon-manual_pred-corr.png")
```

Comment:
```suggestion
        file_name = self.ads_temp_dir / (myelin_mask_overlay.name[:-len("-myelin")] + "-axon-corr.png")
```
Same as above (internal file/overlay naming I think)